assets/js/blocks/featured-items/featured-category/utils.ts

```diff
@@ -1,15 +1,13 @@
 /**
  * External dependencies
  */
+import { WP_REST_API_Category } from 'wp-types';
 import { isObject } from 'lodash';
 
 /**
  * Get the src from a category object, unless null (no image).
- *
- * @param {Object|null} category A product category object from the API.
- * @return {string} The src of the category image.
  */
-function getCategoryImageSrc( category ) {
+export function getCategoryImageSrc( category: WP_REST_API_Category ) {
 	if ( category && isObject( category.image ) ) {
 		return category.image.src;
```

Comment:
Minor: `Property 'src' does not exist on type 'object'.ts(2339)`, same for image.id below.

Reply:
Good catch! 👏